bot/validations.ts

```diff
@@ -259,26 +281,27 @@ const validateBuyOrder = async ctx => {
     return false;
   }
 };
-const validateLightningAddress = async lightningAddress => {
+const validateLightningAddress = async (lightningAddress: string) => {
   const pattern = /^[\w-.]+@([\w-]+.)+[\w-]{2,4}$/g;
 
   return (
     pattern.test(lightningAddress) && existLightningAddress(lightningAddress)
   );
 };
 
-const validateInvoice = async (ctx, lnInvoice) => {
+const validateInvoice = async (ctx: MainContext, lnInvoice: string) => {
   try {
     const invoice = parsePaymentRequest({ request: lnInvoice });
     const latestDate = new Date(
-      Date.now() + parseInt(process.env.INVOICE_EXPIRATION_WINDOW)
+      Date.now() + Number(process.env.INVOICE_EXPIRATION_WINDOW)
     ).toISOString();
-    if (!!invoice.tokens && invoice.tokens < process.env.MIN_PAYMENT_AMT) {
+    if (process.env.MIN_PAYMENT_AMT === undefined) throw Error("MIN_PAYMENT_AMT is undefined, please check .env file");
```

Comment:
@Mersho to avoid using the word `undefined` here (to respect our workflow guidelines), I guess you can do `if ("MAIN_PAYMENT_AMT" in process.env)`?

Reply:
Done